go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/orange-cloudavenue/terraform-provider-cloudavenue
 go 1.20
 
 require (
+	github.com/FrangipaneTeam/terraform-plugin-framework-validators v0.0.0-20230301104912-dc064f6846d6
```

Comment:
Make a PR for specific enhancement.

Reply:
It's not an enhancement is for use the validator in resource.